Porting/config.sh

```diff
@@ -43,13 +43,13 @@ afs='false'
 afsroot='/afs'
 alignbytes='8'
 aphostname=''
-api_revision='5'
-api_subversion='1'
-api_version='33'
-api_versionstring='5.33.1'
+api_revision='7'
+api_subversion='0'
+api_version='1'
```

Comment:
Is there any reason we're keeping these at their "old" names, rather than renaming them to `api_version_{major,minor,patch}` ?

Reply:
Backwards compatibility. This looks like exactly the thing  Perl-Toolchain-Gang/ExtUtils-MakeMaker#358 needed.

Reply:
Yes I would like to tackle these as a following task: patchlevel, subversion, api_*... macros are legacy
I think @ether is right, we want to remove them from core and let Devel-PPPort provide the backward compatibility layer. But this seems to me a follow up task and should not be part of this initial bump to `7.1.0`.